client/client.py

```diff
@@ -44,11 +45,20 @@ async def on_message(self, message):
                         self.db.update_code(message.author, code)
                         await message.author.send("I've sent you a code. Enter it.")
                         print(f"===DEBUG: Generated code: {code}")
+                else:
+                    await message.author.send("Seems that you aren`t registered jet. Enter your school e-mail. I'll send you a confirmation code.")
+                    self.db.add_new_member(message.author)
 
     def generate_code(self):
         """Generate verification code."""
-        # TODO
-        return "123"
+        alphabet = list('1234567890QWERTYUIOPASDFGHJKLZXCVBNM')
+        return "".join(map(lambda x: random.choice(alphabet), (["0"] * 32)))
+
+    @staticmethod
+    def clean_message(msg):
```

Comment:
Why do we need this? I guess, if the message sent contains wrong symbols, such message is wrong and we should ask to enter again.

Reply:
I the text of email, text goes like:
"fishfishfish\n
{code}\n
fishfishfish"
I thought that "\n" can be problem. I always had such issues in university tasks.